src/consumers/mod.rs

```diff
@@ -13,6 +15,11 @@ fn get_credentials() -> (EnvironmentVariableCredentialsProvider, Option<String>)
     (EnvironmentVariableCredentialsProvider::new(), region)
 }
 
+/// Initialize SQS consumers
+///
+/// Initialize 2 SQS consumers:
+/// * Start indexer consumer
+/// * Failed indexer consumer
```

Comment:
should we specify the consumers? if we add more consumers later on we will have to come back and update this comment

Reply:
Yes I think we should update these, the whole process is as we said a bit opaque and it's not clear what it's doing so for now we can specify in comments but I think the structure can be improved.

Reply:
alright